knowledge/responses/i_love_how_animals_taste.md

```diff
@@ -1,7 +1,11 @@
+The question is: “do you value your taste buds higher than the life of an animal”? And it is not only about the taking of the animal's life. To reach your plate and give you a few minutes of pleasure, animals are forcefully in-bred, live horrendous lives of constant abuse, inhabit bodies that no longer work properly due to genetical modification, and are finally taken to death camps where they see their peers being killed one by one. This is all in accordance with "humane" regulations.
 
-I think one of the reasons why as vegans we find this excuse a really difficult one to hear is because it doesn’t matter whatever product people claim they could never give up or enjoy the taste of, the difficulty and the inconvenience that it would be for them to no longer eat animal products is minuscule compared to the pain, suffering and fear that the animals have to go through. A non-vegan meal lasts for 15 minutes, but the death of an animal is eternal. What food could ever be worth taking the life of an individual? An individual that felt fear, an individual that felt pain, an individual that felt confused and couldn’t understand why such intolerable suffering was being committed against them? The fear of death is not unique to humans. Most of us fear dying and hope it will happen peacefully, surrounded by the ones we love without pain. The scariest thing we can think of is being tortured and ultimately murdered. We make horror films about this concept and it gives us nightmares. But this isn’t a horror film for the trillions of animals that are slaughtered each year, it isn’t some nightmare that they will wake from. It’s real, the pain, the fear, it is all real - and then they die and they are gone forever.
+So, if you think that a few minutes of pleasure justifies the immense cruelty these sentient beings are made to suffer, you'd need to think what else can be justified by pleasure alone. Would this argument hold up in court if a murderer in their own defence said, “but I could never give up murdering because I enjoy it too much.”? For instance, in places in the world where women are second class citizens, the way they dress is considered a justification for rape. They are seen as having incited the will for physical pleasure in their assailants.
```

Comment:
```suggestion
If you think that a few minutes of pleasure justifies the immense cruelty these sentient beings are made to suffer, you'd need to think what else can be justified by pleasure alone. Would this argument hold up in court if a murderer in their own defense said that they could never give up murdering because of pleasure? 
```